libs/payments/ui/src/lib/nestapp/validators/DetermineLocationActionResult.ts

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { IsString, ValidateNested } from 'class-validator';
+import { Type } from 'class-transformer';
+
+class LocationInformation {
+  @IsString()
```

Comment:
can we swap this for `@IsEnum(LocationStatus)`? See the `@IsEnum(CartEligibilityStatus)` on the `SetupCartActionResult.ts` validator.